2023-06-07 16:32:13,952 ERROR: Exception on /user/mortimer/doctor/11/edit [POST] [in /home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/home/erica/portfolio/medication_tracker/app/main/routes.py", line 317, in edit_doctor
    return render_template('doctor.html', title="Doctor Information", user=user, doctor=doctor, form=form)
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/templating.py", line 147, in render_template
    return _render(app, template, context)
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/flask/templating.py", line 130, in _render
    rv = template.render(context)
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/erica/portfolio/medication_tracker/app/templates/doctor.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/erica/portfolio/medication_tracker/app/templates/base.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/erica/portfolio/medication_tracker/app/templates/doctor.html", line 40, in block 'content'
    <p>{{ form.delete_confirmation.label }}
  File "/home/erica/portfolio/medication_tracker/venv/lib/python3.10/site-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'app.main.forms.EditDoctorForm object' has no attribute 'delete_confirmation'
